layouts/partials/install.html

```diff
@@ -2,114 +2,108 @@
   <div class="container">
 
     <h2 class="content-center">Install IPFS</h2>
-    <h5 class="content-center" style="margin-bottom: 60px;">No matter how you want to use IPFS, there's an install
-      option for you. Unsure which to choose? Start with <a href="https://github.com/ipfs-shipyard/ipfs-desktop"
-        target="_blank"> IPFS Desktop</a>.</h5>
+    <h5 class="content-center" style="margin-bottom: 60px;">Join the future of the web right now &mdash; just choose the option that's right for you.</h5>
 
+    <h3 class="content-center">Store and share files</h3>
     <div class="grid-flex-container">
-      <div class="grid-flex-cell-1of2">
-        <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank">
+
+      <div class="grid-flex-cell-1of1">
           <div class="install-item">
             <div class="install-item-image">
-              <img src="/images/ipfs-desktop-hex.png" />
+              <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank" rel="noopener noreferrer"><img src="/images/ipfs-desktop-hex.png" /></a>
             </div>
             <div class="install-description">
-              <h4>Desktop app</h4>
+              <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank" rel="noopener noreferrer"><h4>IPFS Desktop</h4></a>
+              <h5>IPFS for everyone</h5>
               <p>
-                IPFS Desktop is an easy-to-use app that enables you to run an IPFS node on your computer without having
-                to
-                bother with command-line tools.
+                The desktop app offers menubar/tray shortcuts and an easy interface for adding, pinning, and sharing files &mdash; plus a full Go install behind the scenes when you're ready to build. A great choice for devs and non-devs alike.
```

Comment:
`plus a full Go install behind the scenes when you're ready to build`

This bit needs a rework:

* We typically use `go-ipfs`... but here I think we should use `full IPFS node` because this is for either end-user or developer.
* `behind the scenes` sounds borderline shady, should replace with something empowering and active. This gives short shrift to the main event.
* `ready to build` makes it sound like a full node is for development, which isn't really the case. I like that you're calling attention to the developer angle though.

Maybe something like: `comes with a full IPFS node ready for heavy duty hosting, and development too`

Reply:
Hadn't thought of "behind the scenes" in that way, so thanks for calling it out. I like your suggested text - done.